README.md

```diff
@@ -54,7 +54,8 @@ LoggingSystem.bootstrap(StreamLogHandler.standardError)
 
 For further information, please check the [API documentation][api-docs].
 
-#### Backends
+#### Selecting a logging backend implementation (applications only)
+<a name="backends"></a>
```

Comment:
You might want to put this before the heading. Otherwise, when the browser scrolls to the anchor, the heading will be above the visible area of the page.

Reply:
Agreed, fixed 👍